.github/workflows/mt-record-screenshots.yml

```
@@ -0,0 +1,255 @@
+          fi
+          echo "MODULE_APK_FILE=$APK_FILE" >> $GITHUB_OUTPUT
+          echo "Module app APK downloaded successfully."
+        env:
+          GH_TOKEN: ${{ secrets.MT_PAT }}
+
+      - name: MT build this app release APK (fallback)
+        if: steps.download-module-apk.outcome == 'failure' || github.event.inputs.buildFromSource == 'true'
+        id: build-module-apk
+        env:
+          MT_ENCRYPT_KEY: ${{ secrets.MT_ENCRYPT_KEY }}
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:22:37: Double quote to prevent globbing and word splitting

(shellcheck)